### ISSUE TITLE
Help wanted

### ISSUE THREAD

**Issue by SiNeumann** [Kirides/vscode-daedalus#1]
Hey,
I saw your comment on my issue [https://github.com/dzieje-khorinis/DaedalusCompiler/issues/58] I just checked your profile and saw you are working on an extension for vscode and make use of the compiler. I started a similar project the last couple days because I didn't know about yours. So I'm currently looking into your projects and if you care I might add some PR's.

**Comment by Kirides** [comment]
Oh, i'm sorry, for some reason i was not *watching* this repository.

I'm totally in for some help. Especially on the vscode side of things.

Most of the work is done in [Kirides/DaedalusLanguageServer](https://github.com/Kirides/DaedalusLanguageServer) where i currently use a modified version of [dzieje-khorinis/DaedalusCompiler](https://github.com/dzieje-khorinis/DaedalusCompiler) to parse the scripts and try to provide meaningful information.

I have to rewrite the code to use the latest compiler though - or rather - i want to do it.

**Comment by SiNeumann** [comment]
Yeah, no worries. I noticed the diff in the g4 file and made some changes on a local branch to get it working on my machine with the dzieje-khorinis/DaedalusCompiler as submodule, which works for me but nothing I'd dare to push, I wanted to wait for 0.8 anyway. 
What I could do on the VsCode side of things are some snippets, some tasks(e.g.build with problemmatchter) and maybe also filetemplates (NPC,DIALOG etc, this would create a dependency on another extension). But well show don't talk I hope the have a PR ready the upcoming weekend

**Comment by pm4rcin** [comment]
@SiNeumann hi, it's been a long time. Are you still interested in adding it?
